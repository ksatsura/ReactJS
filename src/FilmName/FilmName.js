//imports
import React from 'react';
import '../style.css';

const FilmName = (props) => {

    return <p className='film-asset-name'>{props.title}</p>;
};

export default FilmName;