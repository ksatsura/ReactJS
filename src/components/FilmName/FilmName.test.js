import React from 'react';
import FilmName from './FilmName';
import {shallow} from 'enzyme';

describe('FilmName.js', () => {

    const props = {
        title: '',
    };

    describe('renders', () => {

        it('renders for desktop', () => {
            const component = shallow(<FilmName {...props} />);

            expect(component).toMatchSnapshot();
        });
    });
});