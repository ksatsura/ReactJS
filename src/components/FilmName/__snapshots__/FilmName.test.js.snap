// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmName.js renders renders for desktop 1`] = `
<p
  className="film-asset-name"
/>
`;
